vendors/espressif/boards/esp32/ports/ble/nimble/iot_ble_hal_gatt_server.c

```diff
@@ -456,6 +466,7 @@ BTStatus_t prvBTGattServerInit( const BTGattServerCallbacks_t * pxCallbacks )
     BTStatus_t xStatus = eBTStatusSuccess;
 
     ble_hs_cfg.gatts_register_cb = prvGATTRegisterCb;
+    serviceCnt = 0;
```

Comment:
This small change addresses the issue observed when creating services twice, e.g. enabling "BLE_Write_Notification_Size_Greater_Than_MTU_3​" and "BLE_Send_Data_After_Disconnected" at the same time
while running `iot_test_ble_hal_integration`. Previous to this change the valid handle was not getting initialized.
CC: @ravibhagavandas @dhrishi